The nodeadkeys variant of Mac layouts is unusable, modifiers and others
important keys are not defined.

Not submitted yet.

Index: xkb-data/rules/base.mlv_s.part
===================================================================
--- xkb-data.orig/rules/base.mlv_s.part
+++ xkb-data/rules/base.mlv_s.part
@@ -1 +1 @@
- $mac		$macvendorlayouts	nodeadkeys	=	macintosh_vndr/us(extended)+macintosh_vndr/%l%(v)
+ $mac		$macvendorlayouts	nodeadkeys	=	pc(pc105)+macintosh_vndr/us(extended)+macintosh_vndr/%l%(v)
